corsheaders/models.py

```diff
@@ -3,3 +3,7 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+
+    @python2_unicode_compatible
+    def __str__(self)
```

Comment:
also please add a test that just instantiates a `CorsModel` and checks the output of running `str()` on it